src/cloud-api-adaptor/test/provisioner/provision.go

```diff
@@ -368,6 +381,22 @@ func (p *KeyBrokerService) GetKbsEndpoint(ctx context.Context, cfg *envconf.Conf
 	return "", fmt.Errorf("Service %s not found", serviceName)
 }
 
+func (p *KeyBrokerService) EnableKbsAllowAllPolicy(kbsEndpoint string) error {
+	log.Info("EnableKbsAllowAllPolicy")
+	kbsClientDir := filepath.Join(TRUSTEE_REPO_PATH, "target/release")
+	privateKey := "../../kbs/config/kubernetes/base/kbs.key"
+	policyFile := "../../kbs/sample_policies/allow_all.rego"
+	cmd := exec.Command("./kbs-client", "--url", kbsEndpoint, "config", "--auth-private-key", privateKey, "set-resource-policy", "--policy-file", policyFile)
```

Comment:
I don't recommend using the kbs client to provision policies at the moment, those APIs are still somewhat in flux atm.

Can we mount the correct policy as a file using configmap in overlays instead?

Reply:
@mkulke I created issue in trustee here https://github.com/confidential-containers/trustee/issues/380 for this, @fitzthum suggest we use the kbs-client. What's your concern for this tool?

Reply:
ah, there's no fundamental issue, if it works atm, I just thought it would be more straightforward to overwrite the default kbs policy via k8s, since we also orchestrate the kbs deployment.

The CRUD aspects of the KBS API are a bit in flux, because we don't have clear abstractions/semantics for storage everywhere yet, so I wouldn't advertise the tool for resource administration yet.

Reply:
Seen from the change history in https://github.com/confidential-containers/trustee/tree/main/kbs/tools/client, sounds like its stable.

Reply:
ah, the API design might be somewhat stable. I was referring to a lack of explicit state management in kbs in general. At the moment it's just local fs on the container. E.g. if you deploy kbs with a replication of > 1, it will update the policy only in one instance. If the container is moved to another node, the changes are lost, etc.

Atm, we use the default kbs deployment yamls and those set a replication = 1 and hopefully the process doesn't crash, so we should be fine in the test.

Reply:
Got it and agreed, how to set OPA policy in KBS for multiple replicas deployment. 
Which looks like a requirement in trustee repo and should be fixed in kbs client or kbs server. How about we reopen https://github.com/confidential-containers/trustee/issues/380 and track it in trustee repo?

Reply:
maybe it's better to have it handled as part of the kbs operator, I'm not sure. We can keep using kbs client with the restriction in mind.

my idea was to inject policies (and secrets) via a test kustomize overlays that we create:

./overlay/test/kustomization.yaml:
```yaml
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
namespace: coco-tenant

resources:
- ../../base

patches:
- path: patch.yaml
  target:
    group: apps
    kind: Deployment
    name: kbs
    version: v1
```

./overlay/test/opa-policy.yaml:
```yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: opa-policy
data:
  policy.rego: |
    package policy

    default allow = false

    allow {
      input["tee"] == "sample"
    }
```

./overlay/test/patch.yaml:
```yaml
- op: add
  path: /spec/template/spec/volumes/-
  value:
    name: opa-policy
    configMap:
      name: opa-policy
- op: add
  path: /spec/template/spec/containers/0/volumeMounts/-
  value:
    name: opa-policy
    mountPath: /opt/confidential-containers/kbs/policy.rego
    subPath: policy.rego
```

Reply:
@mkulke I'm using `TEE_CUSTOMIZED_OPA` to identify whether a customized OPA is used so that a negative test case can also be added for sample TEE for libvirt provider, what about we use this option for now and considering add the overlay in a separate PR?